Resources/Prototypes/_L5/Recipes/Lathes/devices.yml

```diff
@@ -0,0 +1,10 @@
+- type: latheRecipe
```

Comment:
Infinite hand teleporters is kinda crazy. I think this should be a single item that is mapped in on the nucleus, but I don't feel strongly about it.

Reply:
Realizing now that I'm not sure multiple hand teleporters on the same map work together. Going to remove this.